docs/template_plugin/src/transformations/preprocessing/std_scale.cpp

```diff
@@ -32,10 +32,10 @@ ngraph::pass::AddStdScale::AddStdScale(const ScaleMap& inputInfoMap) {
             "Scale for ", param->get_friendly_name(), " must have f32 type");
 
         auto copy_param = param->clone_with_new_inputs({});
-        auto mul = std::make_shared<ngraph::opset3::Multiply>(copy_param, it->second);
+        auto div = std::make_shared<ngraph::opset3::Multiply>(copy_param, it->second);
```

Comment:
revert back changes in template plugin?

Reply:
Yes, we need it now. I've just reverted it partially to ensure CI is getting green.
I think, we need implement preprocessing on CPU like in another plugins (with multiplicative stdScale) and create ticket for correcting it for all plugins.

Reply:
Done. As was decided, I disabled tests for myriad with inconsistent implementation.